Index: llvm/_core.c
===================================================================
--- llvm/_core.c	(revision 85)
+++ llvm/_core.c	(working copy)
@@ -329,14 +329,17 @@
 _wLLVMValueGetUses(PyObject *self, PyObject *args)
 {
     LLVMValueRef value;
+    LLVMValueRef *uses;
+    PyObject *list;
+    unsigned n;
 
     if (!(value = (LLVMValueRef)get_object_arg(args)))
         return NULL;
 
-    LLVMValueRef *uses = 0;
-    unsigned n = LLVMValueGetUses(value, &uses);
+    uses = 0;
+    n = LLVMValueGetUses(value, &uses);
 
-    PyObject *list = make_list_from_LLVMValueRef_array(uses, n);
+    list = make_list_from_LLVMValueRef_array(uses, n);
     if (n > 0)
         LLVMDisposeValueRefArray(uses);
 
